view/statistics-files-pending.js

```diff
@@ -66,7 +66,7 @@ exports['statistics-main'] = function () {
 	});
 	table({ class: 'statistics-table' },
 		thead(tr(
-			th(),
+			th(_('Date:'), ' ', window.moment().format('DD/MM/YYYY')),
```

Comment:
Please use what we usually use: `new db.Date()`. This way you should get the formatting right for every end system.